frontend/src/fetchers/ssdi.js

```diff
@@ -0,0 +1,51 @@
+import join from 'url-join';
+import { get } from './index';
+import { filtersToQueryString } from '../utils';
+import { QA_DASHBOARD_FILTER_CONFIG } from '../pages/QADashboard/constants';
+
+const ssdiUrl = join('/', 'api', 'ssdi');
+
+const allowedTopicsForQuery = {
+  'recipients-with-no-tta': [
```

Comment:
Not sure if it makes sense to put this in a constants if we will use it in other locations.

Reply:
I don't think this is final until the backend is done, but sure at that point we can move it into a constant